wallet/src/de/schildbach/wallet/ui/CreateUsernameActivity.kt

```diff
@@ -133,7 +135,7 @@ class CreateUsernameActivity : InteractionAwareActivity(), TextWatcher {
     }
 
     private fun initViewModel() {
-        val confirmTransactionSharedViewModel = ViewModelProvider(this).get(SingleActionSharedViewModel::class.java)
+        val confirmTransactionSharedViewModel = ViewModelProvider(this).get(ConfirmTransactionDialog.SharedViewModel::class.java)
```

Comment:
This should be PlatformPaymentConfirmDialog instead of ConfirmTransactionDialog

Reply:
Good find!

Reply:
Fixed